.github/workflows/boost-issues.yml

```diff
@@ -0,0 +1,20 @@
+# Add issues labelled "[Plugin] Boost" to the Jetpack Boost Project Board
+
+name: Add to Jetpack Boost Project board
```

Comment:
Do we think other teams might create project boards and want issues automatically added to them too? If so, it might make sense to name it (both here and the filename) something somewhat more generic.

Reply:
That is a good point! I've renamed it, and updated the naming / comments to support others joining in on this script in c8256a79f.